Cache/IO.lean

```diff
@@ -302,14 +303,16 @@ def packCache (hashMap : HashMap) (overwrite verbose : Bool) (comment : Option S
     let buildPaths ← mkBuildPaths path
     if ← allExist buildPaths then
       if overwrite || !(← zipPath.pathExists) then
+        acc := acc.push (path, zip)
```

Comment:
This was deliberately left out, because the purpose of the "packing {path} as {zip}" logging is to show the files that were actually sent to the server (which currently is all of them, but should only be the new or changed files after this PR).

Reply:
nevermind, misread